src/content/docs/tutorial-errors/respond-outages.mdx

```diff
@@ -0,0 +1,155 @@
+  />
+
+  <TechTile
+    name=".NET agent"
+    icon="logo-dotnet"
+    to="https://one.newrelic.com/nr1-core?state=30e93090-6dfa-6b70-8e75-472f54414355"
+  />
+
+  <TechTile
+    name="Node.js agent"
+    icon="logo-nodejs"
+    to="https://one.newrelic.com/marketplace/install-data-source?state=be2e62fa-cc3b-c428-27c4-8d662c9e80a1"
+  />
+
+  <TechTile
+    name="PHP agent"
+    icon="logo-php"
+    to="https://one.newrelic.com/nr1-core?state=aa633b41-72d4-009c-3abf-55dcf64894fe"
+  />
+
+  <TechTile
+    name="Python agent"
+    icon="logo-python"
+    to="https://one.newrelic.com/nr1-core?state=20fda75b-58fb-a92a-f9e1-7b052035c6e8"
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=d69143ab-605c-579b-25bf-cc6e5fee5b80"
+  />
+
+</TechTileGrid>
+
+Once you've installed an agent, go to **[one.newrelic.com](https://one.newrelic.com/nr1-core?filters=(domain%3D'APM'ANDtype%3D'APPLICATION'))** and select your app. If you don't see much data just yet, step away for a while and let the agent gather real-time data as your application runs.
+
+## Find the critical error [#critical-error]
+
+Now that your apps are instrumented, New Relic is capturing data about your services. This includes data about error occurrences in your app. 
+
+<Steps>
+    <Step>
+        ### Think about the end user
+
+Alerts let you know a problem exists: they're the water on your lawn. But alerts won't provide you all the context. That's where your errors inbox comes in. 
+
+Imagine you're responsible for some apps on an ecommerce site. You've received two alerts for two components, one for checking out and one for searching inventory. You're only receiving reports that the search functionality is failing for end users, but the check out component works fine. You may believe that the check out function is more important, but it's critical to separate your beliefs from your observability practices. 
+
+This practice applies even if the end user hasn't reported anything. When you notice services failing, you can ask yourself these questions:
+
+* Is the end user experiencing a problem? 
+* How should their experience look?
+* What behavior are they currently experiencing?
+
+    </Step>
+    <Step>
+        ### Choose a service
+
+Let's see how this might look in practice. When you view the **All entities** page, you notice four services are alerting.
+
+<img
+title="Overview errors affecting your services"
+alt="A screenshot showing an app with many errors"
+src={allEntitiesView}
+/>
+
+After asking yourself the questions from step one, you know that: 
+
+* The end user is struggling with purchase actions.
+* Your site should only display in-stock items.
+* Your site is displaying all products, so customers are able to purchase out-of-stock items.
+
+After performing your audit, you know that `api-gateway` is a critical dependency for your inventory. This is where you'll begin your error analysis. 
```

Comment:
I'm not sure we really established that an "audit" occurred earlier in the doc. (This is the first instance of the word `audit` in the doc.) I think this sentence could use more foreshadowing earlier.